### DISCUSSION TITLE
Check-group should replace btn-group here

### DISCUSSION THREAD

**Discussion by ZhangChengLin** [twbs/bootstrap#31218] [General]
https://v5.getbootstrap.com/docs/5.0/forms/checks/#radio-toggle-buttons

Should btn-group here be named check-group? 

PS: The label elements inside are visible, but they don’t seem to be arranged with equal width?

**Comment by patrickhlauke** [comment]
no, it intentionally only uses `.btn-group` as a generic grouping class. and no, it doesn't try to make them equal width - that's something that authors may have to deal with separately.

incidentally, i have a PR in progress in this area https://github.com/twbs/bootstrap/pull/31201 (see the preview links at the end of the PR description)

**Reply by ZhangChengLin** [reply]
> and no, it doesn't try to make them equal width - that's something that authors may have to deal with separately.

Seeing your contribution, they are great.

Do you have examples of making them medium-width in a row?
In the style class within bootstrap.
